packages/flutter_tools/lib/src/run.dart

```diff
@@ -0,0 +1,338 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
Can `flutter drive` reuse this code? There seems to be quite a bit of duplication.